plugins/modules/rocketchat.py

```diff
@@ -104,10 +104,9 @@
     description:
       - If V(true), the payload matches Rocket.Chat prior to 7.4.0 format. This format has been used by the module since its
         inception, but is no longer supported by Rocket.Chat 7.4.0.
-      - The default value of the option is going to change to V(false) eventually.
+      - The default value of the option, V(true), is **deprecated** since community.general 11.2.0 and will change to V(false) in community.general 13.0.0.
```

Comment:
```suggestion
      - The default value of the option, V(true), is B(deprecated) since community.general 11.2.0 and will change to V(false) in community.general 13.0.0.
```

Reply:
:facepalm: thank you!